### PR TITLE
Translations update from Hosted Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Jamulus/Jamulus app](https://hosted.weblate.org/projects/jamulus/jamulus-app/).

It also includes following components:

* [Jamulus/Windows Installer](https://hosted.weblate.org/projects/jamulus/windows-installer/)

Current translation status:

Weblate translation status

### CODE REVIEW

File: src/translation/translation_pt_BR.ts [hunk 1]
```diff
@@ -376,7 +376,7 @@
     <message>
         <location filename="../audiomixerboard.cpp" line="254"/>
         <source>Grp</source>
-        <translation>Grp</translation>
+        <translation></translation>
```
Comment (ann0see) [inline src/translation/translation_pt_BR.ts:379]: I don't that's ok.

**Comment by ann0see** [comment]
This should be merged ASAP.

File: src/translation/translation_de_DE.ts [hunk 2]
```diff
@@ -3547,7 +3547,7 @@ Wir haben Deinen Kanal stummgeschaltet und die Funktion &apos;Stummschalten&apos
     </message>
     <message>
         <source>Or select &apos;%1&apos; and specify a Custom Directory address on the Options tab to register with a custom directory.</source>
-        <translation type="unfinished">Oder wähle &quot;%1&quot; und gebe in der Registerkarte &quot;Optionen&quot; eine benutzerdefinierte Verzeichnisadresse an, um dich bei einem benutzerdefinierten Verzeichnis zu registrieren.</translation>
+        <translation>Oder wählen Sie &quot;%1&quot; und geben Sie auf der Registerkarte &quot;Optionen&quot; eine benutzerdefinierte Verzeichnisadresse an, um sich bei einem benutzerdefinierten Verzeichnis zu registrieren.</translation>
```
Comment (ann0see) [inline src/translation/translation_de_DE.ts:3550]: Sie --> du

File: src/translation/translation_de_DE.ts [hunk 3]
```diff
@@ -147,7 +147,7 @@
     <message>
         <location filename="../util.cpp" line="745"/>
         <source>Norwegian Bokmål</source>
-        <translation type="unfinished"></translation>
+        <translation>Norwegischer Bokmal</translation>
```
Comment (ann0see) [inline src/translation/translation_de_DE.ts:150]: Not sure if that's correct. Norwegischer --> Norwegisch

File: src/translation/translation_de_DE.ts [hunk 4]
```diff
@@ -3886,7 +3886,7 @@ Wir haben Deinen Kanal stummgeschaltet und die Funktion &apos;Stummschalten&apos
...
         <comment>%1: directory type NONE; %2: directory type CUSTOM; %3 app name, Jamulus</comment>
-        <translation type="unfinished"></translation>
+        <translation>&lt;b&gt;Directory:&lt;/b&gt; Select &apos;%1&apos;um Ihren Server nicht bei einem Verzeichnis zu registrieren. &lt;br&gt;Oder wählen Sie eines der Verzeichnisse aus, um sich bei diesem Verzeichnis zu registrieren. &lt;br&gt;Oder wählen Sie &quot;%2&quot; und geben Sie auf der Registerkarte Optionen eine benutzerdefinierte Verzeichnisadresse an, um sich bei einem benutzerdefinierten Verzeichnis zu registrieren. &lt;br&gt;&lt;br&gt;Für jeden Wert außer &apos;%1&apos; registriert sich dieser Server bei einem Verzeichnis, so dass ein %3-Benutzer diesen Server in der Serverliste des Client-Verbindungsdialogs auswählen kann, wenn er dieses Verzeichnis wählt. &lt;br&gt;&lt;br&gt;Die Registrierung des Servers wird in regelmäßigen Abständen erneuert, um sicherzustellen, dass alle Server in der Serverliste des Verbindungsdialogs tatsächlich verfügbar sind.</translation>
```
Comment (ann0see) [inline src/translation/translation_de_DE.ts:3889]: Not sure if that's correct...

File: src/translation/translation_de_DE.ts (hunk 3 above)
Comment (ann0see) [reply src/translation/translation_de_DE.ts:150]: Resolved on Weblate

File: src/translation/translation_de_DE.ts (hunk 4 above)
Comment (ann0see) [reply src/translation/translation_de_DE.ts:3889]: Resolved on Weblate

File: src/translation/translation_es_ES.ts [hunk 5]
```diff
@@ -3946,7 +3946,7 @@ Hemos silenciado tu canal y activado &apos;Silenciarme Yo&apos;. Por favor resue
...
         <comment>%1: directory type NONE; %2: directory type CUSTOM; %3 app name, Jamulus</comment>
-        <translation type="unfinished"></translation>
+        <translation>&lt;b&gt;Directorio:&lt;/b&gt; Selecciona &apos;%1&apos; para no registrar tu servidor con un directorio.&lt;br&gt;O selecciona uno de los géneros para registrarlo con ese directorio.&lt;br&gt;O selecciona &apos;%2&apos; y especifica una dirección de Directorio Personalizada en la pestaña de Opciones para registrarlo con un directorio personalizado.&lt;br&gt;&lt;br&gt;Para cualquier valor excepto &apos;%1&apos;, este servidor se registra con un directorio para que un usuario de %3 pueda escoger este servidor de la lista de servidores en la ventana de conexión del cliente, donde escoge ese directorio.&lt;br&gt;&lt;br&gt;El registro del servidor se renueva periódicamente para asegurarse de que todos los servidores de la lista de servidores en la ventana de conexión realmente están disponibles.</translation>
```
Comment (ann0see) [inline src/translation/translation_es_ES.ts:3949]: @ignotus666 I think you wrote this, please review and approve.

File: src/translation/translation_fr_FR.ts [hunk 6]
```diff
@@ -3080,7 +3080,7 @@ Nous avons coupé votre canal et activé &quot;Me silencer&quot;. Veuillez d&apo
...
         <comment>%1: directory type NONE; %2: directory type CUSTOM; %3 app name, Jamulus</comment>
-        <translation type="unfinished"></translation>
+        <translation>&lt;b&gt;Annuaire :&lt;/b&gt; sélectionnez &apos;%1&apos; pour ne pas enregistrer votre serveur avec un annuaire.&lt;br&gt;Ou sélectionnez l&apos;un des genres pour vous enregistrer avec cet annuaire.&lt;br&gt;Ou sélectionnez &apos;%2&apos; et spécifiez une adresse d&apos;annuaire personnalisé dans l&apos;onglet Options pour vous enregistrer avec un annuaire personnalisé.&lt;br&gt;&lt;br&gt;Pour toute valeur sauf &apos;%1&apos;, ce serveur s&apos;enregistre auprès d&apos;un annuaire afin qu&apos;un utilisateur %3 puisse sélectionner ce serveur dans la liste des serveurs du dialogue de connexion du client lorsqu&apos;il choisit cet annuaire.&lt;br&gt;&lt;br&gt;L&apos;enregistrement du serveur est renouvelé périodiquement pour s&apos;assurer que tous les serveurs de la liste des serveurs du dialogue de connexion sont réellement disponibles.</translation>
```
Comment (ann0see) [inline src/translation/translation_fr_FR.ts:3083]: @trebmuh @jujudusud please review (and approve if everything is ok)

File: src/translation/translation_pl_PL.ts [hunk 7]
```diff
@@ -3123,7 +3123,7 @@ Twój kanał został wyciszony i włączono „Wycisz mnie”. Napraw przyczynę
...
         <comment>%1: directory type NONE; %2: directory type CUSTOM; %3 app name, Jamulus</comment>
-        <translation type="unfinished"></translation>
+        <translation>&lt;b&gt;Katalog:&lt;/b&gt;Wybierz &apos;%1&apos;, aby nie rejestrować serwera w katalogu.&lt;br&gt;lub wybierz jeden z gatunków, aby zarejestrować się w tym katalogu.&lt;br&gt;lub wybierz &apos;%2&apos; i podaj adres Niestandardowy katalog w zakładce Opcje, aby zarejestrować się w niestandardowym katalogu.&lt;br&gt;&lt;br&gt;Dla każdej wartości oprócz &apos;%1&apos;, ten serwer rejestruje się z katalogiem, aby użytkownik %3 mógł wybrać ten serwer na liście serwerów okna dialogowego client connect, gdy wybiera ten katalog.&lt;br&gt;&lt;br&gt;Rejestracja serwera jest odnawiana okresowo, aby upewnić się, że wszystkie serwery na liście serwerów okna dialogowego connect są rzeczywiście dostępne.</translation>
```
Comment (ann0see) [inline src/translation/translation_pl_PL.ts:3126]: @SeeLook please review and approve if everything is OK.

File: src/translation/translation_pt_BR.ts [hunk 8]
```diff
@@ -199,7 +199,7 @@
         <location filename="../aboutdlgbase.ui" line="72"/>
         <source>Copyright (C) 2005-2022 The Jamulus Development Team</source>
         <translatorcomment>I&apos;ll keep it untranslated for the moment</translatorcomment>
-        <translation></translation>
+        <translation>Copyright (C) 2005-2022 A equipe de desenvolvimento Jamulus</translation>
```
Comment (ann0see) [inline src/translation/translation_pt_BR.ts:202]: @melcon @Snayler please review and approve if everything is ok.

File: src/translation/translation_zh_CN.ts [hunk 9]
```diff
@@ -2606,7 +2606,7 @@ We muted your channel and activated &apos;Mute Myself&apos;. Please solve the fe
...
         <source>&lt;b&gt;Directory:&lt;/b&gt; Select &apos;%1&apos; not to register your server with a directory.&lt;br&gt;Or select one of the genres to register with that directory.&lt;br&gt;Or select &apos;%2&apos; and specify a Custom Directory address on the Options tab to register with a custom directory.&lt;br&gt;&lt;br&gt;For any value except &apos;%1&apos;, this server registers with a directory so that a %3 user can select this server in the client connect dialog server list when they choose that directory.&lt;br&gt;&lt;br&gt;The registration of the server is renewed periodically to make sure that all servers in the connect dialog server list are actually available.</source>
         <comment>%1: directory type NONE; %2: directory type CUSTOM; %3 app name, Jamulus</comment>
-        <translation type="unfinished"></translation>
+        <translation>&lt;b&gt;目录：&lt;/b&gt;选择 &apos;%1&apos; 不在目录中注册你的服务器。&lt;br&gt;或者选择其中一个流派在该目录中注册。&lt;br&gt;或者选择 &apos;%2&apos; 并在选项标签上指定一个自定义目录地址，在一个自定义目录中注册。&lt;br&gt;&lt;br&gt;对于除&apos;%1&apos;以外的任何值，该服务器都会与一个目录注册，这样，当 %3 的用户选择该目录时，可以在客户端连接对话服务器列表中选择该服务器。&lt;br&gt;&lt;br&gt;服务器的注册会定期更新，以确保连接对话服务器列表中的所有服务器都是实际可用的。</translation>
```
Comment (ann0see) [inline src/translation/translation_zh_CN.ts:2609]: @BLumia please review and approve if everything is ok.

**Review by ann0see — APPROVED** [review]
German is "ok" although the du/sie inconsistency isn't clear yet. I think we try to avoid it in general.

**Review by ignotus666 — APPROVED** [review]

File: src/translation/translation_zh_CN.ts (hunk 9 above)
Comment (BLumia) [reply src/translation/translation_zh_CN.ts:2609]: Looks good to me :)

**Review by BLumia — APPROVED** [review]

File: src/translation/translation_fr_FR.ts (hunk 6 above)
Comment (trebmuh) [reply src/translation/translation_fr_FR.ts:3083]: LGTM

**Review by trebmuh — APPROVED** [review]

**Comment by ann0see** [comment]
Merging as active translators have approved it.

**Pull request merged**